api-specifications/admin-api/admin-api-1.yaml

```diff
@@ -0,0 +1,1355 @@
+openapi: 3.0.1
+info:
+  title: Admin API Documentation
+  version: v1
```

Comment:
I would put the full API specification version number here. Suggest change from "v1" to "1.4.0" so to have the full semantic version.